README.md

```diff
@@ -54,6 +54,19 @@ call the method `Datadog::Statsd#flush` if you want to force the sending of metr
   * using singletons instances of the DogStatsD client and not allocating one each time you need one, letting the buffering mechanism flush metrics, or,
   * properly closing your DogStatsD client instance when it is not needed anymore using the method `Datadog::Statsd#close` to release the resources used by the instance and to close the socket
 
+### Known incompatibility with v5.x
```

Comment:
These aren't technically incompatibilities. Maybe have the title be "v5.0 Common Pitfalls" or something like that?

Reply:
Good advice, pushing this change.